geometry/Crossing.cpp

```diff
@@ -226,6 +227,17 @@ int Crossing::GetPartialDoorUsage() const
      return _partialDoorUsage;
 }
 
+void Crossing::ResetDoorUsage()
+{
+     _doorUsage = 0;
+
+     if ((_outflowRate >= std::numeric_limits<double>::max())){
```

Comment:
you mean `<`?

Reply:
No, the door flow should only be closed, if no outflowRate set but a maxDoorUsage